### DISCUSSION TITLE
频繁出现Account_nonce_precondition_unsatisfied

### DISCUSSION THREAD

**Discussion by lispking** [openbuildxyz/mina-zkapp-bootcamp#428] [Q&A]
是什么原因引起？试了好几次依然会出现该问题？该如何解决该问题？

<img width="892" alt="image" src="https://github.com/user-attachments/assets/ad95b6c0-f487-432a-94f0-f4cd22b0cf0c" />

**Comment by MartinYeung5** [comment]
我也遇到相同情況, 在想是不是可能跟合約更新有關
